.travis.yml

```diff
@@ -26,7 +26,34 @@ after_script:
   - if [[ $TRAVIS_PHP_VERSION = '7.0' && $DB = 'sqlite' ]]; then php ocular.phar code-coverage:upload --format=php-clover build/logs/clover.xml; fi
 
 matrix:
+  fast_finish: true
   include:
+    - php: hhvm-3.15
```

Comment:
Yes and no...

I chose [3.15 since it's LTS](http://hhvm.com/blog/6083/hhvm-long-term-support), I believe the next LTS will be 3.18 or 3.20, or maybe 4.0. Considering 3.15 like an LTS php version 5.6 or 7.0

To be on the actual latest of HHVM we can skip specifying the version, but we can't put it in the base configuration since the Trusty container builds on Travis CI have not been released, and the precise containers only support HHVM 3.9. [See HHVM options](https://docs.travis-ci.com/user/languages/php#HHVM-versions)

So it's just a matter of do you only want to test latest, LTS versions, both, etc. 

We can also add builds for [HHVM in php 7 mode](https://docs.hhvm.com/hhvm/configuration/INI-settings#php-7-settings) rather than the default php 5 mode. (I skipped that option since [HHVM has an issue with the PHP 7 mode](https://github.com/facebook/hhvm/issues/7198) that is blocking composer from working)